src/validation.h

```diff
@@ -881,6 +879,9 @@ class ChainstateManager
      */
     RecursiveMutex& GetMutex() const LOCK_RETURNED(::cs_main) { return ::cs_main; }
 
+    //! If the tip is older than this, the node is considered to be in initial block download.
```

Comment:
In commit "Move ::nMaxTipAge into ChainstateManager" (fa6744c42e8bafd2769dd90e4b00af92e5dd4fe0)

If rebasing on #25905, would be nice to move this comment to the options struct to preserve it

Reply:
thx, rebased